nextflow.config

```
@@ -220,17 +220,26 @@ profiles {
             ]
         }
     }
+    gpu {
+        docker.runOptions       = '-u $(id -u):$(id -g) --gpus all'
+        apptainer.runOptions    = '--nv'
+        singularity.runOptions  = '--nv'
+    }
     test          { includeConfig 'conf/test.config'          }
     test_controls { includeConfig 'conf/test_controls.config' }
     test_full     { includeConfig 'conf/test_full.config'     }
 }
 
-// Load nf-core custom profiles from different Institutions
-includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
+// Load nf-core custom profiles from different institutions
+
+// If params.custom_config_base is set AND either the NXF_OFFLINE environment variable is not set or params.custom_config_base is a local path, the nfcore_custom.config file from the specified base path is included.
+// Load nf-core/atacseq custom profiles from different institutions.
+includeConfig params.custom_config_base && (!System.getenv('NXF_OFFLINE') || !params.custom_config_base.startsWith('http')) ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
+
 
 // Load nf-core/atacseq custom profiles from different institutions.
 // TODO nf-core: Optionally, you can add a pipeline-specific nf-core config at https://github.com/nf-core/configs
-// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/atacseq.config" : "/dev/null"
+// includeConfig params.custom_config_base && (!System.getenv('NXF_OFFLINE') || !params.custom_config_base.startsWith('http')) ? "${params.custom_config_base}/pipeline/atacseq.config" : "/dev/null"
```

This should be uncommented
```suggestion
includeConfig params.custom_config_base && (!System.getenv('NXF_OFFLINE') || !params.custom_config_base.startsWith('http')) ? "${params.custom_config_base}/pipeline/atacseq.config" : "/dev/null"
```